### PR TITLE
fix: several deprecated code

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

File: packages/@o3r/dev-tools/src/cli/artifact-cleaner.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ import * as winston from 'winston';
 
 program
   .description('Clean old artifacts from artifactory repositories')
-  .option('--artifactory-url <artifactoryUrl>', 'Artifactory URL', /.*/, 'https://digitalforairlines.jfrog.io/digitalforairlines/')
-  .option('-d, --duration-kept <durationKept>', 'All artifacts which have not been downloaded and are older than this value(ms) will be deleted. Default to 10080000ms (7 days)', /.*/, '604800000')
+  .option('--artifactory-url <artifactoryUrl>', 'Artifactory URL', 'https://digitalforairlines.jfrog.io/digitalforairlines/')
```
Comment (cpaulve-1A) [inline packages/@o3r/dev-tools/src/cli/artifact-cleaner.ts:12]: does it make sense to keep this default value?

Comment (kpanot) [reply packages/@o3r/dev-tools/src/cli/artifact-cleaner.ts:12]: I did not think much than removing deprecated function call.
If so maybe a clean up of several CLIs options can be done in separated task

**Comment by guilhermevrs** [comment]
Still no tests - I am not surprised @kpanot

**Comment by kpanot** [comment]
> Still no tests - I am not surprised @kpanot 

This is because this PR is not breaking test that there is no test in the diff ;)

**Review by matthieu-crouzet — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**